assembl/static2/css/bootstrap-overrides.scss

```diff
@@ -4,3 +4,20 @@
   box-shadow: none !important;
   height: $navbar-height !important;
 }
+
+.modal-content {
```

Comment:
Yes... I put bootstrap overrides in the same file and I would like to find a better solution. This file is destined to disappear ASAP.